Dockerfile

```diff
@@ -6,6 +6,6 @@ RUN apt-get -y update && apt-get install -y -q build-essential
 ADD jars jars
 ADD consumer.properties.template .
 ADD run_kcl.sh .
-ADD kinesis-to-firehose kinesis-to-firehose
+ADD kinesis-consumer kinesis-consumer
```

Comment:
nit: do you think it would be more clear if each app produced a binary named after the current app, and this line was

```
ADD kinesis-to-firehose kinesis-consumer
```

Because that makes life simpler with the `consumer.properties` file?